internal/cli/cmd/hubble.go

```diff
@@ -93,11 +93,6 @@ func newCmdHubbleEnable() *cobra.Command {
 
 	for flagName := range hubble.FlagsToHelmOpts {
 		// TODO(aanm) Do not mark the flags has deprecated for now.
```

Comment:
Do we still need this `TODO`?

Reply:
not sure, i just didn't want to remove somebody else's TODOs. let's see what @aanm says ✨

Reply:
@michi-covalent @viktor-kurchenko if we are deprecating `--helm-set` then we should warn users about it, which is what the commented code is for. So we can remove the TODO as long we uncomment the following code.

Reply:
alright then i'll keep the TODO for now ✅

Reply:
The TODO needs to be kept alongside the code that was removed :sweat_smile:

Reply:
you mean the code that was commented out? sure i can put it back